src/assets/scss/components/event.scss

```diff
@@ -102,6 +112,12 @@ $black-shadow: rgba(0, 0, 0, .5);
             color: map-get($map: $event-palette, $key: active);
         }
 
+        &.group {
+            .event-message {
+                padding-left: rem-calc(60px);
```

Comment:
tohle by se mohlo spočítat ne ? šířka ikonky + odsazení ?

Reply:
zatim jsem neprisel na to jak bez js

Reply:
dáš si šířku ikony do proměnné tedy $icon-size: 32px; , pak $icon-offset
a v padding-left : $icon-size + $icon-offset;